enzyme/test/Enzyme/ReverseMode/blas/cblas_daxpy_a.ll

```diff
@@ -0,0 +1,26 @@
+;RUN: %opt < %s %loadEnzyme -enzyme -mem2reg -instsimplify -simplifycfg -S | FileCheck %s
+
+define void @wrapper(i32 %n, double %a, double* %x, double* %y) {
+entry:
+  tail call void @cblas_daxpy(i32 %n, double %a, double* %x, i32 1, double* %y, i32 1)
+  ret void
+}
```

Comment:
Can you make a test where x and/or y are overwritten after the blas call (to check caching works)